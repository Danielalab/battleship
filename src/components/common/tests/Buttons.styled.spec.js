import 'jest-styled-components';
import { render } from '@testing-library/react';
import { Button, ButtonSuccess } from '../Buttons.styled';

describe('Buttons', () => {
  test('Should renders Button component', () => {
    const { container } = render(<Button />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  test('Should renders ButtonSuccess component', () => {
    const { container } = render(<ButtonSuccess />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
