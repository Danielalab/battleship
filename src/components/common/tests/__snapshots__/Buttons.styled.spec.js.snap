// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons Should renders Button component 1`] = `
.c0 {
  cursor: pointer;
  font-size: 1.5rem;
  font-family: 'Press Start 2P',cursive;
  margin: 0.75rem;
  padding: 0.5rem 1rem;
  text-transform: uppercase;
}

<div>
  <button
    class="c0"
  />
</div>
`;

exports[`Buttons Should renders ButtonSuccess component 1`] = `
.c0 {
  cursor: pointer;
  font-size: 1.5rem;
  font-family: 'Press Start 2P',cursive;
  margin: 0.75rem;
  padding: 0.5rem 1rem;
  text-transform: uppercase;
}

.c1 {
  background-color: #113CFC;
  color: #ffffff;
}

.c1:disabled {
  background-color: transparent;
  color: #ffffffa2;
}

<div>
  <button
    class="c0 c1"
  />
</div>
`;
